roles/pulp-devel/tasks/install_podman.yml

```diff
@@ -28,4 +28,16 @@
   args:
     creates: /etc/containers
   when: ansible_distribution == 'Debian'
+
+- name: Enable unprivileged_userns_clone (Debian specific)
+  sysctl:
+    name: kernel.unprivileged_userns_clone
+    value: "1"
+  when: ansible_distribution == 'Debian'
+
+- name: Set max_user_namespaces (CentOS specific)
+  sysctl:
+    name: user.max_user_namespaces
+    value: "15000"
+  when: ansible_distribution == 'CentOS'
```

Comment:
We should have comments about why we are touching these system settings.

Reply:
I will add some comments.

Reply:
Comments added. :wink: